buildSrc/src/main/kotlin/publishing-convention.gradle.kts

```diff
@@ -0,0 +1,63 @@
+import org.gradle.kotlin.dsl.`maven-publish`
+import org.gradle.kotlin.dsl.apply
+import org.gradle.kotlin.dsl.configure
+import org.gradle.kotlin.dsl.signing
+import org.gradle.kotlin.dsl.withType
+
+plugins {
+    `maven-publish`
+    signing
+    id("pl.allegro.tech.build.axion-release")
+}
+
+apply(from = "../versionConfig.gradle")
+
+group = LibConfig.group
+version = scmVersion.version
+
+val javadocJar: TaskProvider<Jar> = tasks.register("javadocJar", Jar::class.java) {
+    archiveClassifier.set("javadoc")
+}
+
+configure<PublishingExtension> {
+    publishing {
+        publications.withType<MavenPublication> {
+            artifact(javadocJar)
+
+            pom {
+                name.set(LibConfig.name)
+                description.set("Kotlin multiplatform JsonLogic expressions evaluation engine")
+                url.set(LibConfig.repositoryUrl)
+                inceptionYear.set("2022")
+                licenses {
+                    license {
+                        name.set("The Apache License, Version 2.0")
+                        url.set("https://www.apache.org/licenses/LICENSE-2.0.txt")
+                    }
+                }
+                developers {
+                    developer {
+                        name.set("Marek Krogulski")
```

Comment:
we should do something about it :D DO we have a task for it?

Reply:
I would add it to the **clean up before the 1.0 release** task. I dont remember any specific task for the developers in pom file